kratos/utilities/embedded_skin_utility.cpp

```diff
@@ -39,8 +39,8 @@ namespace Kratos
 
         // Auxiliar vectors to store pointers to the new entities
         // These vectors will be use when renumbering the entities ids.
-        ModelPart::NodesContainerType new_nodes_vect;
-        ModelPart::ConditionsContainerType new_conds_vect;
+        std::vector<Node::Pointer> new_nodes_vect;
```

Comment:
this looks ok to me but @rubenzorrilla i think this is your code, can you take a look